routes/members.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const router = express.Router()
+const membersController = require('../controllers/membersController')
+
+router.get('/', membersController.getMembers)
+
+/**
+ * @swagger
```

Comment:
Add the annotations above the route initialization.

Reply:
Done!